modules/terragrunt/init_test.go

```diff
@@ -16,10 +15,8 @@ func TestTerragruntStackInit(t *testing.T) {
 	require.NoError(t, err)
```

Comment:
I think we also need a test for error case, to track that it is catchd during init